lib/resources/r4/procedure.yaml

```diff
@@ -43,7 +42,7 @@ fields:
 
 - name: code
   required: 'Yes'
-  cardinality: 0..1
+  required: 'No'
```

Comment:
duped required keys

Reply:
[Removed the dupe and some missed cardinalities](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-07c686ec4248977e084ba5a602c761ceL85)